src/ripple/rpc/handlers/DepositAuthorized.cpp

```diff
@@ -102,7 +102,7 @@ doDepositAuthorized(RPC::JsonContext& context)
         {
             // See if a preauthorization entry is in the ledger.
             auto const sleDepositAuth =
-                ledger->readSLE(keylet::depositPreauth(dstAcct, srcAcct));
+                ledger->read(keylet::depositPreauth(dstAcct, srcAcct));
             depositAuthorized = static_cast<bool>(sleDepositAuth);
```

Comment:
I'd be inclined to change these lines into...
```
depositAuthorized =
    ledger->read(keylet::depositPreauth(dstAcct, srcAcct))
        .has_value();
```
That way you don't even need to name a local variable.

Reply:
great point 👍